projects/packages/blaze/src/class-dashboard.php

```diff
@@ -0,0 +1,179 @@
+<?php
+/**
+ * A class that adds a Blaze dashboard to wp-admin.
+ *
+ * @since $$next-version$$
+ *
+ * @package automattic/jetpack-blaze
+ */
+
+namespace Automattic\Jetpack\Blaze;
+
+use Automattic\Jetpack\Assets;
+
+/**
+ * Responsible for adding a Blaze dashboard menu to wp-admin.
+ * The screen content itself is rendered remotely.
+ */
+class Dashboard {
+	/**
+	 * Package version.
+	 *
+	 * @var string
+	 */
+	const PACKAGE_VERSION = '0.6.0-alpha';
+
+	/**
+	 * List of dependencies needed to render the dashboard in wp-admin.
+	 *
+	 * @var array
+	 */
+	const JS_DEPENDENCIES = array( 'lodash', 'react', 'react-dom', 'wp-components', 'wp-compose', 'wp-i18n', 'wp-is-shallow-equal', 'wp-primitives', 'wp-url', 'moment' );
+
+	/**
+	 * URL where we load the Blaze dashboard remotely.
+	 *
+	 * @var string
+	 */
+	const CDN_URL = 'https://widgets.wp.com/blaze-dashboard/%s/%s';
+
+	/**
+	 * Script handle for the JS file we enqueue in the dashboard page.
+	 *
+	 * @var string
+	 */
+	const SCRIPT_HANDLE = 'jetpack-blaze-dashboard';
+
+	/**
+	 * We bump the asset version when the Jetpack back end is not compatible anymore.
+	 *
+	 * @var string
+	 */
+	const BLAZEDASH_VERSION = 'v1';
+
+	/**
+	 * Cache key for the cache buster.
+	 *
+	 * @var string
+	 */
+	const BLAZEDASH_CACHE_BUSTER_CACHE_KEY = 'jetpack_blaze_admin_asset_cache_buster';
+
+	/**
+	 * Override render funtion
+	 *
+	 * @return void
+	 */
+	public function render() {
+		?>
+		<div id="wpcom" class="jp-blaze-dashboard" style="min-height: calc(100vh - 100px);">
+			<div class="hide-if-js"><?php esc_html_e( 'Your Jetpack Blaze dashboard requires JavaScript to function properly.', 'jetpack-blaze' ); ?></div>
+			<div class="hide-if-no-js" style="height: 100%">
+				<img
+					class="jp-blaze-dashboard-loading-spinner"
+					width="32"
+					height="32"
+					style="position: absolute; left: 50%; top: 50%;"
+					alt=<?php echo esc_attr( __( 'Loading', 'jetpack-blaze' ) ); ?>
+					src="//en.wordpress.com/i/loading/loading-64.gif"
+				/>
+			</div>
+		</div>
+		<script>
+			jQuery(document).ready(function($) {
+				// Load SVG sprite.
+				$.get("https://widgets.wp.com/blaze-dashboard/common/gridicons-506499ddac13811fee8e.svg", function(data) {
```

Comment:
This one doesn't load for me yet, I get a 404.

Reply:
Yes, this file will be included on the widgets CDN. I am already working on pushing the first version of the app there. 
I will let you know when I have the phabricator patch so you can test the integration like in production.